commands/commands_mock_test.go

```diff
@@ -6,11 +6,15 @@ import (
 	"github.com/mesg-foundation/core/commands/provider"
 	"github.com/mesg-foundation/core/container"
 	"github.com/mesg-foundation/core/protobuf/coreapi"
-	"github.com/mesg-foundation/core/service"
 	"github.com/mesg-foundation/core/utils/servicetemplate"
 	"github.com/stretchr/testify/mock"
 )
 
+var (
+	_ RootExecutor    = (*mockRootExecutor)(nil)
+	_ ServiceExecutor = (*mockServiceExecutor)(nil)
```

Comment:
I don't see why you added these 2 variables that are not used anywhere

Reply:
it's to make sure to check if types implements the interfaces correctly or not in the compiler time